src/styles/variables.scss

```diff
@@ -0,0 +1,14 @@
+$header-logo-link-size: 40px;
+$header-link-color: #060b35;
+$header-link-active-color: #00acdc;
+$card-width: 200px;
+$card-img-width: 160px;
+$card-background-color: #fff;
+$card-border-color: #f3f3f3;
+$card-img-height: 134px;
+$card-title-color: #060b35;
+$card-subtitle-color: #616070;
+$card-button-height: 40px;
+$card-button-color: #00acdc;
+$card-link-color: #fff;
+$card-title-hover-color:#34568b;
```

Comment:
most of them are useless, you use them only once, don't create variables only for the sake of it